src/Kunstmaan/TranslatorBundle/Controller/TranslatorController.php

```diff
@@ -115,15 +117,15 @@ public function addAction(Request $request, $keyword = '', $domain = '', $locale
 
                 return new RedirectResponse($this->generateUrl(
                     $indexUrl['path'],
-                    isset($indexUrl['params']) ? $indexUrl['params'] : array()
+                    isset($indexUrl['params']) ? $indexUrl['params'] : []
                 ));
             }
         }
 
-        return array(
-            'form' => $form->createView(),
-            'adminlistconfigurator' => $configurator
-        );
+        return [
+            'form'                  => $form->createView(),
```

Comment:
The indentation does not need to change here. PHP coding standards don't say anything about indenting the various "=>" assignment operators. This is a Zend coding standard.